src/box/txn.c

```diff
@@ -1193,6 +1177,18 @@ txn_commit(struct txn *txn)
 	return -1;
 }
 
+int
+txn_commit_submit(struct txn *txn)
```

Comment:
Do you think we should expose `box_txn_commit_submit` to the public C API as well?

Reply:
Nope. Only if somebody asks. If you think this is needed, then I can do it.

Reply:
Ok, let's wait for people's request.